en/docs/guides/usecases/order-management-automation.md

```
@@ -0,0 +1,429 @@
+
+## Run and verify
+
+1. Select **Run** on the integration overview. The first run needs the database password. When prompted, create the `Config.toml` and add:
+
+    ```toml
+    ordersDBPassword = "orders_pass"
+    ```
+
+2. Watch the terminal. Each waiting order advances, and a final line reports the count:
+
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)